package.json

```diff
@@ -16,13 +18,8 @@
     "eslintconfig"
   ],
   "files": [
-    "browser-config.js",
     "index.js",
-    "lit-config.js",
-    "node-config.js",
-    "react-config.js",
-    "sort-member-config.js",
-    "testing-config.js"
+    "utils.js"
```

Comment:
I think you will need to list all the files here, otherwise they won't end up in the NPM package. A shortcut could be to move them into a `configs` directory or something and then just list `/configs` here.